.npmrc

```diff
@@ -0,0 +1,2 @@
+@dimensiondev:registry=https://npm.pkg.github.com/
+//npm.pkg.github.com/DimensionDev:_authToken=1bbc5896d56bf4135a1a9f9fd0ecfbcbb70a0977
```

Comment:
What is this token used for? Is it safe to commit to the git?

Reply:
https://github.com/orgs/DimensionDev/teams/dev/discussions/6

Reply:
safe